ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,155 +1,169 @@
 {# eos - VLAN Interfaces #}
-{% if vlan_interfaces is defined and vlan_interfaces is not none %}
+{% if vlan_interfaces is arista.avd.defined %}
```

Comment:
This check is included in natural_sort, so we no longer need to have this if outside the for-loops, as long as we use natural_sort.